modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -321,6 +272,25 @@ class="jetpack-search-filters-widget__search-box-enabled"
 				</label>
 			</p>
 
+			<div class="jetpack-search-filters-widget__post-types-select">
+				<p class="jetpack-search">
+					<label><?php esc_html_e( 'Post types included in results:' ); ?></label>
+					<select
+						class="widefat jetpack-search-filters-widget__post-type-selector"
+						name="<?php echo esc_attr( $this->get_field_name( 'post_types' ) ); ?>[]"
+						multiple="multiple">
+						<?php foreach ( get_post_types( array( 'exclude_from_search' => false ), 'objects' ) as $post_type ) : ?>
```

Comment:
This is just a heads-up, we may want to do some hacking/exceptions of which post types are searchable later, since some plugins (I'm looking at you, bbPress) set post-types as non-searchable only outside the context of their own UI (e.g. forum search) when in fact they _are_ searched by the plugin's code. It seems like it would be desirable at some point to be able to search for these items.